test/test_edit_all_mountings_and_quotas.py

```diff
@@ -0,0 +1,80 @@
+'''
+    test_edit_all_mountings_and_quotas.py test the page view of Edit All Mountings and Quotas
+'''
+from paste.fixture import TestApp
+from nose.tools import assert_equal
+from app import APP
+from components import session
+
+
+class TestCode(object):
+    '''
+        This class runs the test cases to test Edit All Mountings and Quotas page
+    '''
+    SELECT_MODULE_LABEL = '<b>Select module(s) to edit:'
+    SELECT_MODULE_INPUT = '<input type="text" id="select-mod-to-edit" ' +\
+                          'class="typeahead" data-provide="typeahead">'
+
+    TABLE_HEADER_CODE = '<th>Code</th>'
+    TABLE_HEADER_NAME = '<th>Name</th>'
+    TABLE_HEADER_MOUNTED_IN_SEM1 = '<th>Mounted In Sem 1</th>'
+    TABLE_HEADER_SEM1_QUOTA = '<th>Sem 1 Quota</th>'
+    TABLE_HEADER_MOUNTED_IN_SEM2 = '<th>Mounted In Sem 2</th>'
+    TABLE_HEADER_SEM2_QUOTA = '<th>Sem 2 Quota</th>'
+    TABLE_HEADER_REMOVE = '<th>Remove</th>'
+
+    BUTTON_SAVE_CHANGES = '<button type="submit" class="btn btn-primary edit-all-bottom-button" ' +\
+                          'data-toggle="tooltip" data-placement="bottom" title="Save changes" '
+    BUTTON_RESET_TABLE = '<button type="button" class="btn btn-primary edit-all-bottom-button" ' +\
+                         'data-toggle="tooltip" data-placement="bottom" title="Reset table" '
+
+
+    def __init__(self):
+        self.middleware = None
+        self.test_app = None
+
+
+    def setUp(self):
+        '''
+            Sets up the 'app.py' fixture
+        '''
+        self.middleware = []
+        self.test_app = TestApp(APP.wsgifunc(*self.middleware))
+        session.set_up(self.test_app)
+
+
+    def tearDown(self):
+        '''
+            Tears down 'app.py' fixture and logs out
+        '''
+        session.tear_down(self.test_app)
+
```

Comment:
You might want to also include test cases where certain buttons are clicked. (e.g. the Save button, and Reset Table button, etc).

See if there are any ways to test those 2 buttons with [paste](https://chromium.googlesource.com/external/trace-viewer/+/bf55211014397cf0ebcd9e7090de1c4f84fc3ac0/third_party/Paste/paste/fixture.py). If this cannot be done, then this test case is alright as it is.